docs/1___overview.adoc

```diff
@@ -181,18 +181,7 @@ The following changes were performed as part of the 2.0.0 major release:
   ** Clarify uniqueness of root SSDs in https://github.com/modelica/ssp-standard/pull/23
   ** Clarify hierarchical parameter name resolution in https://github.com/modelica/ssp-standard/pull/27
 
-=== Changes in 1.0.1
 
-The following changes were performed as part of the 1.0.1 maintenance release:
 
-* Correct version string for 1.0 release (Issue #54)
-* Clarify that GTypeChoice is optional for connectors (Issue #80)
-* Clarify use of binary connectors (Issue #98)
-* Clarify interaction with structured variable naming convention in FMI (Issue #59)
-* Clarify icon rotation specification (Issue #29)
-* Add non-normative comment on empty/missing source attribute of components (Issue #82)
-* Clarify connector geometry override (Issue #30)
-* Add further examples of graphical notation (Issue #42)
-* Minor typographical fixes (Issues #56, #57, #58)
 
-Note that there were no changes to the XML Schema files of the standard.
```

Comment:
We might want to retain the 1.0.1 changes here, as some of them point to important points for implementor consideration. But could go either way.